### PR TITLE
COG-1315-Account locked changelog status

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1297

#### Brief feature description
- Changelog status for locked users

### CODE REVIEW

File: app/javascript/_utils/formatters.test.js [hunk 1]
```diff
@@ -187,3 +195,29 @@ describe('#formatPermissions', () => {
...
+    expect(formatMadeByEvent(adminName, adminRole)).toEqual('Some Admin')
+  })
+
+  it('returns only adminName when adminRole is undefined', () => {
+    const adminName = 'Some Admin'
+    const adminRole = undefined
+    expect(formatMadeByEvent(adminName, adminRole)).toEqual('Some Admin')
+  })
+
+  it('returns only adminName when adminRole is null', () => {
```
Comment (Nasir2712) [inline app/javascript/_utils/formatters.test.js]: what if adminName and adminRole are undefined?

Comment (sunilb0575) [reply app/javascript/_utils/formatters.test.js]: It will return undefined.

Comment (kelfink) [reply app/javascript/_utils/formatters.test.js]: Would be nice to issue a test for that.  Also better if it wasn't undefined.  Maybe empty string.  Something we can display without thinking too much about it.

Comment (sunilb0575) [reply app/javascript/_utils/formatters.test.js]: @kelfink Right now, we are not checking any of the events for null or undefined. I think a new user story should be created in order to determine what to be shown on UI when null or undefined.

Comment (kelfink) [reply app/javascript/_utils/formatters.test.js]: Sounds like good technical debt, if you write it up.

Comment (sunilb0575) [reply app/javascript/_utils/formatters.test.js]: Sure.

**Pull request merged**